src/build/build_helpers.jl

```diff
@@ -179,11 +178,11 @@ function get_tax_revenue_for_consumer(jm, m, cr::ConsumerRef)
             for tax in output.taxes
                 if cr.subindex === nothing
                     if get_full(tax.agent) == get_full(cr)    
-                        push!(taxes, :($(tax.rate) * $(output.quantity) * $(output.commodity) * $(pf.sector)))
+                        push!(taxes, :($(tax.rate) * $jm[get_comp_supply_name($output)] * $(output.commodity) * $(pf.sector)))
```

Comment:
Are you sure this works? I think what would happen here is that the value of `jm` gets interpolated into the expression, and then the expression continues with `[get_comp_supply_name(`. But that is presumably not what we want? I.e. we would want `get_comp_supply_name` to evaluate while we build the expression, right? I would think it needs to be `$(jm[get_comp_supply_name(output)])`. Same for the next diff below.

Reply:
It _seems_ to work either way, but I updated as above.

Reply:
Yeah, it might work, but the new way should be more efficient, I think :)